docs/data/data-grid/components/components.md

```diff
@@ -212,6 +212,10 @@ This demo below shows how to use the `slotProps` prop and module augmentation to
 
 {{"demo": "CustomFooter.js", "bg": "inline"}}
 
+## 🚧 Using Joy UI components
```

Comment:
Would a generic section like `Choose design system` and sub-sections `Material UI` and `Joy UI` make more sense, to let users be more familiar with switching between the two?

Reply:
This is a good idea to implement when we have the Joy UI integration ready and working, but before that we would likely keep this demo hidden

Reply:
I moved it to a separate page that will not be deployed to production for now: https://deploy-preview-8067--material-ui-x.netlify.app/x/react-data-grid/experiments/joy-ui/